mingw-w64-vsg/PKGBUILD

```
@@ -0,0 +1,75 @@
+# Maintainer: Konstantin Podsvirov <konstantin@podsvirov.pro>
+
+_fullname=VulkanSceneGraph
+_realname=vsg
```

```suggestion
_realname=vulkanscenegraph
```
see: https://aur.archlinux.org/packages/vulkanscenegraph

---

Overview:
- [vulkanscenegraph](https://repology.org/project/vulkanscenegraph/versions) (AUR)
- [vsg](https://repology.org/project/vsg/versions) (vcpkg, conan)
- [libvsg](https://salsa.debian.org/games-team/vulkanscenegraph/-/blob/master/debian/control?ref_type=heads) (debian)

The CMake package named `vsg` (target `vsg::vsg`).
Header files in `vsg` subfolder.
All classes in `vsg` namespace.

Why we can't use short name as in many cases?

---

We follow Arch naming in general.